src/Sylius/Bundle/ApiBundle/Resources/config/serialization/Adjustment.xml

```diff
@@ -40,6 +40,7 @@
             <group>sylius:shop:adjustment:show</group>
             <group>shop:cart:show</group>
             <group>sylius:shop:cart:show</group>
+            <group>sylius:admin:order_item:index</group>
```

Comment:
I cannot see the `admin:order_item:index` group that reflects `sylius:...` one

Reply:
https://github.com/Sylius/Sylius/commit/fe0ff72d2b147c624ec684f287cacb9cdc2a6ab1#diff-c1535686d2960f7ce6d393db914601f622d3cdaea71087c6c9b70c6eac38bd80R76-R84